monitor/all-scenarios.js

```diff
@@ -5,8 +5,11 @@ const { Worker } = require('worker_threads');
 
 const filesPath = path.join(__dirname, '..', 'scenarios');
 
+const fileToName = (file) => file.replace(/-/g, '.').replace('.js', '').toLowerCase();
```

Comment:
nit-pick: This could go in a helper file since it is reused in multiple files

Reply:
Yeah, I thought about that, but I'm not sure where it would go. Let me think more about how we should structure all of this monitoring code.